packages/safe-apps-provider/src/provider.ts

```diff
@@ -64,6 +64,18 @@ export class SafeAppProvider extends EventEmitter implements EIP1193Provider {
         return '0x';
       }
 
+      case 'eth_signTypedData_v4': {
```

Comment:
let's also add support for `eth_signTypedData`, version postfix is metamask specific and not a part of the official eip